spec/features/admin_adds_and_watches_video_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+feature "Admin adds and watches video" do
+  scenario "admin successfully adds and watches a new video" do
+    admin = Fabricate(:admin)
+    Fabricate(:category, name: "Action")
+    sign_in(admin)
+    visit new_admin_video_path
+
+    fill_in "Title", with: "Rambo"
+    select 'Action', from: 'Category'
+    fill_in "Description", with: "A very fast action movie"
+    attach_file('video[large_cover]', File.join(Rails.root, '/spec/support/uploads/large_ponyo.jpg'))
```

Comment:
Line is too long. [101/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.